src/NzbDrone.Core/CustomFormats/Specifications/ReleaseTitleSpecification.cs

```diff
@@ -8,7 +8,7 @@ public class ReleaseTitleSpecification : RegexSpecificationBase
     {
         public override int Order => 1;
         public override string ImplementationName => "Release Title";
-        public override string InfoLink => "https://github.com/Radarr/Radarr/wiki/Custom-Formats-Aphrodite#release-title";
+        public override string InfoLink => "https://wiki.servarr.com/Radarr_Settings#Custom_Formats_2";
```

Comment:
Why is it `Custom_Formats_2` is there a `?

Reply:
wiki thing; `custom_formats` is the naming schema custom format... this is due to switching to UI based wiki pages